index.html

```diff
@@ -21,6 +21,21 @@
             <div class="header_menu">
                 <img class="menu_icon" src="images/Union.png" alt="menu" />
             </div>
+            
+            
+            <ul class="header_nav_desktop_version">
+                <li>
+                    <a class="black" href="#">Portfolio</a>
+                </li>
+                <li>
+                    <a class="black" href="#">About</a>
+                </li>
+                <li>
+                    <a class="black" href="#">Contact</a>
+                </li>
+            </ul>
+        
+        
         </nav>
     </header>
     <section class="headline">
```

Comment:
- [ ] For the `headline` section, kindly make your design look identical to that of the Figma template you chose. To make it look identical, kindly work on:

- The background images used
- The `hx` elements with texts `"Hey there. I’m Cansu"` and `"I'm a software developer"`
- The dropdown icon used 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208150819-8a5a17a8-0726-4660-893d-5f7dae6a3eec.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/208151065-5ab557b1-341a-412b-9d28-ebf834569bdd.png)